tags/environmentalsustainability.md

```diff
@@ -0,0 +1,77 @@
+# CNCF TAG Environmental Sustainability
+
+## Introduction
+
+The Paris Climate Accords outline the link between economic activity, greenhouse gas (GHG) emissions, and the impacts of climate change. GHG  emissions, primarily made from Carbon Dioxide, are released during the combustion of fossil fuels to produce electricity. To reach net zero goals, we need to identify how to reduce the environmental footprints of increasingly large data centers.
+
+## Mission Statement
+This TAG's goal is to advocate for, develop, support, and help evaluate environmental sustainability initiatives in Cloud Native Technologies.
+
+All activities of this TAG should aim to positively impact the community by changing mindset, increasing resource efficiency, and putting energy efficiency and environmental sustainability on everyone's agenda.
+
+## Responsibilities & Deliverables
+
+### Background
+We recognize that: 
+- balancing resource consumption with the desire for performance is difficult
```

Comment:
`Trade off between reducing resource consumption and higher performance is hard` << is this better?

Reply:
Could you detail what you prefer in your sentence over the original?

Reply:
Easier to follow I think.  as a compromise:
the trade off between reducing resource consumption and higher performance is hard to balance

(same PR as above)

Reply:
first part is an apple ("resource consumption") the second part is an orange ("desire for performance")

Reply:
Problem is, people want apples and oranges on the same box.

Reply:
@catblade "lowering resource consumption" and "higher performance" is comparable ... the problem is the "desire for"

Reply:
Fair enough.  Will update in the other PR.  Thanks for clarifying.